src/style.css

```diff
@@ -0,0 +1,52 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  width: 15px;
+  height: 15px;
+  margin: 0 2px;
+  background: url(./images/star.svg) no-repeat 1px 2px;
+}
+
+.star:nth-child(1) {
+  margin-left: 0;
+}
+
+.star:nth-child(5) {
+  margin-right: 0;
+}
+
+.stars--1 .star:nth-child(1) {
+  background: url(./images/star-active.svg) no-repeat 1px 2px;
+}
+
+.stars--2 .star:nth-child(1),
+.stars--2 .star:nth-child(2) {
+  background: url(./images/star-active.svg) no-repeat 1px 2px;
+}
+
+.stars--3 .star:nth-child(1),
+.stars--3 .star:nth-child(2),
+.stars--3 .star:nth-child(3) {
+  background: url(./images/star-active.svg) no-repeat 1px 2px;
+}
+
+.stars--4 .star:nth-child(1),
+.stars--4 .star:nth-child(2),
+.stars--4 .star:nth-child(3),
+.stars--4 .star:nth-child(4) {
+  background: url(./images/star-active.svg) no-repeat 1px 2px;
+}
+
+.stars--5 .star:nth-child(1),
+.stars--5 .star:nth-child(2),
+.stars--5 .star:nth-child(3),
+.stars--5 .star:nth-child(4),
+.stars--5 .star:nth-child(5) {
+  background: url(./images/star-active.svg) no-repeat 1px 2px;
+}
```

Comment:
you could simplify this

Reply:
I am so thankful that you showed me more elegant way!